libs/blocks/marquee/marquee.css

```diff
@@ -320,6 +340,12 @@
     max-height: 80px;
   }
 
+  .marquee .text .icon-area.icon-area-multiple img {
```

Comment:
Check all of these. I don't think all of the nesting is necessary and starts to impact performance. All these need to be checked.

```suggestion
  .marquee .icon-area-multiple img {
```